pyproject.toml

```diff
@@ -5,14 +5,17 @@ description = ""
 authors = ["meooow25 <soumiksarkar.3120@gmail.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.8"
```

Comment:
what's the exact reason for upgrading python here? if my understanding is not flawed then you essentially are using gemini which wouldn't require updating the python version itself?

Reply:
Gemini does require Python version 3.9+.
[Prerequisites for Gemini](https://ai.google.dev/tutorials/python_quickstart#prerequisites)